### PR TITLE
Add readiness and liveness probes

### PR DESCRIPTION
**Pull request by aleerizw-zz**
**What this PR does / why we need it**:
Adds readiness and liveness probes for jaeger-collector

### CODE REVIEW

**Comment by aleerizw-zz** [comment]
@pavelnikolov @dvonthenen Can you please review

**Comment by pavelnikolov** [comment]
lgtm

File: incubator/jaeger/values.yaml [hunk 1]
```diff
@@ -132,6 +132,7 @@ collector:
     httpPort: 14268
     # can accept Zipkin spans in JSON or Thrift
     zipkinPort: 9411
+    healthCheckPort: 14269
```
Comment (pavelnikolov) [inline incubator/jaeger/values.yaml:135]: Maybe we need to start ordering the port names in alphabetical order

**Review by pavelnikolov — APPROVED** [review]

**Comment by davidvonthenen** [comment]
Yup. This is pretty straightforward. Looks good to me!

**Pull request merged**